examples/step-3/doc/intro.dox

```diff
@@ -91,8 +91,15 @@ the expansion coefficients $U_i$ mentioned above, so that
 @f}
 Note that we here follow the convention that everything is counted starting at
 zero, as common in C and C++. This equation can be rewritten as a linear
-system by inserting the representation $u_h(\mathbf x)=\sum_j U_j
-\varphi_j(\mathbf x)$: Find a vector $U$ so that
+system if you insert the representation $u_h(\mathbf x)=\sum_j U_j
+\varphi_j(\mathbf x)$ and then observe that
+$
+  (\nabla\varphi_i, \nabla u_h)
+= \left(\nabla\varphi_i, \nabla \left[\sum_j U_j \varphi_j\right] \right)
+= \sum_j \left(\nabla\varphi_i, \nabla \left[U_j \varphi_j\right]
+\right)
+= \sum_j \left(\nabla\varphi_i, \nabla \varphi_j \right) U_j
+$. With this, the problem reads: Find a vector $U$ so that
```

Comment:
Did you mean for this to be an inline equation?

Reply:
It looked ok -- I think it looked longer in the latex source than it really is. But on second thoughts, I think it's worthwhile offsetting.